src/main/java/com/mineboundteam/minebound/magic/UtilitySpells/ShieldUtilitySpell.java

```diff
@@ -109,9 +114,10 @@ public void inventoryTick(ItemStack pStack, Level pLevel, Entity pEntity, int pS
         // Add NBT data to item when it first enters the players inventory
         if (!pStack.hasTag() && !pLevel.isClientSide()) {
             CompoundTag spellTag = new CompoundTag();
-            spellTag.putFloat("hits_remaining", totalHits);
-            spellTag.putInt("cooldown", -1);
-            pStack.getOrCreateTag().put("minebound.shield_utility", spellTag);
+            spellTag.putFloat(HITS_TAG, config.TOTAL_HITS.get());
+            spellTag.putInt(COOLDOWN_TAG, -1);
+            spellTag.putInt(MAX_TAG, -1);
+            pStack.getOrCreateTag().put(SHIELD_TAG, spellTag);
         }
```

Comment:
Could you test if we even need this function? No point in maintaining it if we don't.

Reply:
When I was last making changes to shield we did, but I can check again

Reply:
We still need it; nothing happens if we dont' set it

Reply:
I see why now; it's because we don't save the nbt the for the armoritem the first time after editing it. Or at least I think it is..

Reply:
It works with that line, I don't feel like adding an additional serializer just for this spell